pymoskito/__init__.py

```diff
@@ -49,3 +49,5 @@
 register_simulation_module(Disturbance, GaussianNoise)
 
 register_processing_module(PostProcessingModule, PlotAll)
+
+os.environ["QT_AUTO_SCREEN_SCALE_FACTOR"] = "1"
```

Comment:
Since I'm not sure that this setting will work for all use cases, I'd would rather give a hint in the docs and move it to the top of the '__main__.py' of each project.